### PR TITLE
queue source/tests, gitignore, travis config, license, readme, mongo … 

### PR DESCRIPTION
**Pull request by yonderblue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,60 @@
...
+Developers may be contacted at:
+
+ * [Email](mailto:tol_igroup@dominionenterprises.com)
+ * [Pull Requests](https://github.com/dominionenterprises/mongo-queue-php/pulls)
+ * [Issues](https://github.com/dominionenterprises/mongo-queue-php/issues)
+
+##Tests
+
+With a checkout of the code make sure [xbuild](http://www.mono-project.com/Microsoft.Build)
+and [nunit-console](http://www.nunit.org/index.php?p=nunit-console&r=2.2.10) is in your PATH and run:
```
Comment (nubs) [inline README.md:56]: "are" in your PATH.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,60 @@
...
+```
+
+##Build
+
+To add the library as a local, per-project dependency make sure [xbuild](http://www.mono-project.com/Microsoft.Build) is in your PATH and run:
+
+```bash
+xbuild DominionEnterprises.Mongo/DominionEnterprises.Mongo.csproj
+```
+and use the resulting `DominionEnterprises.Mongo/bin/Debug/DominionEnterprises.Mongo.dll` in your project.
```
Comment (nubs) [inline README.md:39]: blank line before this please.

File: README.md (hunk 1 above)
Comment (yonderblue) [inline README.md:56]: k

File: README.md (hunk 2 above)
Comment (yonderblue) [inline README.md:39]: k

**Comment by yonderblue** [comment]
will be adding code coverage check for the build. also dont forget to update the repo desc on DE

**Comment by nubs** [comment]
already done dood.

**Comment by yonderblue** [comment]
sweet :)

**Pull request merged**